Content.Server/Atmos/EntitySystems/GasAnalyzerSystem.cs

```diff
@@ -81,7 +81,10 @@ private void ActivateAnalyzer(EntityUid uid, GasAnalyzerComponent component, Ent
         {
             component.Target = target;
             component.User = user;
-            component.LastPosition = Transform(target ?? user).Coordinates;
+            if (target != null)
+                component.LastPosition = Transform(target ?? default).Coordinates;
```

Comment:
what's with the ?? here if we know target is not null

Reply:
Without the `?? default`:

```
space-station-14/Content.Server/Atmos/EntitySystems/GasAnalyzerSystem.cs(85,52):
  error CS1503: Argument 1: cannot convert from 'Robust.Shared.GameObjects.EntityUid?' to 'Robust.Shared.GameObjects.EntityUid'
```

This was put in place to convince the compiler that it's a suitable argument. I'm new to C#. Is there a more idiomatic way of convincing the compiler that we already know it's not nullable at this point? It doesn't seem to be able to infer based on the previous `if` statement.

Reply:
Use target.Value

Reply:
Should be fixed now. Thank you.